internal/demo/demo.go

```diff
@@ -49,6 +49,9 @@ var fiveEPlayDemoNameRegex = regexp.MustCompile(`^g\d+-(.*)[a-zA-Z0-9_]*$`)
 // https://shobhit-pathak.github.io/MatchZy/gotv/#recording-demos
 var matchZyDemoNameRegex = regexp.MustCompile(`^(\d{4}-\d{2}-\d{2}_\d{2}-\d{2}-\d{2})_(\d+)_([a-zA-Z0-9_]+)_(.+?)_vs_(.+)$`)
 
+// ID of 12 characters long, starts with "n" (probably the game id) and then 11 letters.
```

Comment:
It's a 64-bit integer containing a timestamp, encoded into letters with the alphabet "bcdfghjkmnpqrtvwBCDFGHJKMNPQRTVW". The first letter will become "p" in May. 12 characters should hold true until 2028 then it will be 13 characters.

Reply:
thanks for the info!
could you ask the admins of Esplay to include "Esplay" in the name of the game server?
it's a more common and easier way to identify where a demo comes from.
thanks

Reply:
All future matches will have server name containing "Esplay" now 👍

Reply:
thanks!